src/assets/data/edition/series/1/section/5/op25/graph.json

```diff
@@ -68,21 +68,21 @@
B, 10B‒12/\" ;\n\t awg:precedes awg:M_322_Sk4e .\n\nawg:M_322_Sk4e a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. 13B‒16/\" ;\n\t awg:precedes awg:M_322_Sk4f .\n\nawg:M_322_Sk4f a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. 13C, 14B, {15B}/\" ;\n\t awg:precedes awg:M_322_Sk4g .\n\nawg:M_322_Sk4g a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. 17A‒20A/\" ;\n\t awg:precedes awg:M_322_Sk4h .\n\nawg:M_322_Sk4h a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. {21A}, 21B/\" ;\n\t awg:precedes awg:M_322_Sk4i .\n\nawg:M_322_Sk4i a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. 17B‒20B/\" ;\n\t awg:precedes awg:M_322_Sk4j .\n\nawg:M_322_Sk4j a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t rdfs:label /\"M 322 Sk4 T. 21C/\" .\n\nawg:M_322_Sk4_1 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.1/\" .\n\nawg:M_322_Sk4_2 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.2/\" .\n\nawg:M_322_Sk4_3 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.3/\" .\n\nawg:M_322_Sk4_4 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.4/\" .\n\nawg:M_322_Sk4_5 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.5/\" .\n\nawg:M_322_Sk4_6 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.6/\" .\n\nawg:M_322_Sk4_7 a awg:Sketch ;\n\t dc:isPartOf awg:Op25_2 ;\n\t awg:concomitates awg:M_322_Sk4 ;\n\t rdfs:label /\"M 322 Sk4.7/\" .\n\n# Paratexts SB4 (only dates)\nawg:PT_SB4_2r_1_date_19340822 a awg:Paratext ;\n\t rdfs:label /\"Datierung: 22. VIII. 34/\" ;\n\t awg:concomitates awg:M_321_Sk1 ; \n\t awg:precedes awg:PT_SB4_6r_1_date_19340919 ;\n\t dc:isPartOf awg:Op25_3 .\n\nawg:PT_SB4_6r_1_date_19340919 a awg:Paratext ;\n\t rdfs:label /\"Datierung: 19. IX. 34/\" ;\n\t awg:concomitates awg:M_321_Sk2 ; \n\t awg:precedes awg:PT_SB4_7v_12_date_19341008 ;\n ;\n\t dc:isPartOf awg:Op25_3 .\n\nawg:PT_SB4_7v_12_date_19341008 a awg:Paratext ;\n\t rdfs:label /\"Schlussdatierung: 8. X. 34/\" ;\n\t awg:precedes awg:PT_SB4_8v_4_date_19341024 ;\n\t dc:isPartOf awg:Op25_3 .\n\nawg:PT_SB4_7v_16_date_19340930 a awg:Paratext ;\n\t rdfs:label /\"Datierung: M. am 30. IX. nach Zürich abgereist./\" ; \n\t awg:concomitates awg:M_321_Sk8j , awg:M_321_Sk8k ;\n\t dc:isPartOf awg:Op25_3 .\n\nawg:PT_SB4_8v_4_date_19341024 a awg:Paratext ;\n\t rdfs:label /\"Datierung: 24. X[.] 34/\" ;\n\t awg:concomitates awg:M_322_Sk3a ;\n\t awg:precedes awg:PT_SB4_11r_9_date_19341115 ;\n\t dc:isPartOf awg:Op25_2 .\n\nawg:PT_SB4_11r_9_date_19341115 a awg:Paratext ;\n\t rdfs:label /\"Schlussdatierung: 15. XI[.] 34/\" ;\n\t dc:isPartOf awg:Op25_2 .\n"
```

Comment:
In den Triples bitte alle

`/\"`

durch 

`\"`

ersetzen

Reply:
In dem Triple 

```ttl
awg:PT_SB4_6r_1_date_19340919 a awg:Paratext ;
	 rdfs:label "Datierung: 19. IX. 34" ;
	 awg:concomitates awg:M_321_Sk2 ; 
	 awg:precedes awg:PT_SB4_7v_12_date_19341008 ;
 ;   # dieses Semikolon entfernen
	 dc:isPartOf awg:Op25_3 .
```

gibt es ein unnötiges Semikolon, d.h. in `;\n ;\n\t` das erste `;\n ` entfernen